LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest

```diff
@@ -1,29 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="4c570677-8476-4d33-bd0c-da36c89287c8" Version="2.0.2" Language="en-US" Publisher="rubicon IT GmbH" />
-    <DisplayName>License Header Manager</DisplayName>
-    <Description xml:space="preserve">An extension for adding license headers to source code files.</Description>
-    <MoreInfo>https://github.com/rubicon-oss/LicenseHeaderManager</MoreInfo>
-    <License>VsixInformation/MIT.txt</License>
-    <GettingStartedGuide>https://github.com/rubicon-oss/LicenseHeaderManager/wiki</GettingStartedGuide>
-    <Icon>VsixInformation/Icon.png</Icon>
-    <PreviewImage>VsixInformation/Preview.png</PreviewImage>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[12.0, 15.0]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0, 15.0]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[14.0, 15.0]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Premium" Version="[11.0, 12.0]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Ultimate" Version="[11.0, 12.0]" />
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.0,)" />
-  </Dependencies>
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
-  </Prerequisites>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
-  </Assets>
+    <Metadata>
+        <Identity Id="4c570677-8476-4d33-bd0c-da36c89287c8" Version="2.0.2" Language="en-US" Publisher="rubicon IT GmbH" />
+        <DisplayName>License Header Manager</DisplayName>
+        <Description xml:space="preserve">An extension for adding license headers to source code files.</Description>
+        <MoreInfo>https://github.com/rubicon-oss/LicenseHeaderManager</MoreInfo>
+        <License>VsixInformation/MIT.txt</License>
+        <GettingStartedGuide>https://github.com/rubicon-oss/LicenseHeaderManager/wiki</GettingStartedGuide>
+        <Icon>VsixInformation/Icon.png</Icon>
+        <PreviewImage>VsixInformation/Preview.png</PreviewImage>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[12.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[14.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Premium" Version="[11.0, 12.0]" />
```

Comment:
Why is it necessary to change this file? If i interpret https://blogs.msdn.microsoft.com/visualstudio/2013/08/08/update-for-extension-authors-vsix-manifest-version-range-attribute/ correctly, we seem to cover the same range of Versions (assuming that Minor Versions are not relevant, which they seem not to, as i have no problems installing or finding the License Header Manager extension on VS 15.4.0). 

Also please do remove the unnecessary indents

Reply:
In that file it says: "] – maximum version inclusive." That means to me that it includes 15.0 but nothing beyond that. Now they may have special logic to allow minor numbers beyond the configured maximum of 15.0, but "16.0)" more clearly states the intent. But I can change it back if you want.

As for the indents, I just used the Visual Studio designer to save the file. I think it makes sense to leave the indent the way the tool writes it. I have my XML tab settings set to Insert Spaces and size 2, so I think the tool is just insisting on that. But again, I can revert the whole file if you insist.

Reply:
Changed the rest of them as well.